### PR TITLE
Add tests for generateCartCounterBadge

### PR DESCRIPTION
**Pull request by kschmidtdev**
Tiny PR with some tests.

## Changes
- Tests for `generateCartCounterBadge`

## How to test-drive this PR
- Run the tests
- Observe they pass

### CODE REVIEW

File: app/containers/header/container.test.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    test('Should not be visible with no items in the cart', () => {
+        const result = generateCartCounterBadge({summary_count: 0})
+        expect(result.type).not.toBe(Badge)
+    })
+
+    test('Should not be visible if passed incomplete object', () => {
+        const result = generateCartCounterBadge()
+        expect(result.type).not.toBe(Badge)
+    })
+
```
Comment (DanielBrookRoberge) [inline app/containers/header/container.test.js]: There should probably also be a test for if it is passed a count of 0.

File: app/containers/header/container.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    test('Should be visible and accurate if there are items in the cart', () => {
+        const oneItemResult = generateCartCounterBadge({summary_count: 1})
+        expect(oneItemResult.type).toBe(Badge)
+        expect(oneItemResult.props.children).toBe(1)
+
+        const twoItemsResult = generateCartCounterBadge({summary_count: 2})
+        expect(twoItemsResult.type).toBe(Badge)
+        expect(twoItemsResult.props.children).toBe(2)
```
Comment (DanielBrookRoberge) [inline app/containers/header/container.test.js]: Maybe also check if the title of the Badge contains the count?

File: app/containers/header/container.test.js (hunk 1 above)
Comment (kschmidtdev) [inline app/containers/header/container.test.js]: I think the first test covers that?

Comment (DanielBrookRoberge) [inline app/containers/header/container.test.js]: Oops yeah it does.

**Pull request merged**